### PR TITLE
inplace set to False in extract_nth functions

### PR DESCRIPTION
**Pull request by emlynjdavies**
fixes issue #58

### CODE REVIEW

**Review by tankefugl — APPROVED** [review]
No linting violations have been found in this PR.

File: tfl_tools/silcam_classifier_database_builder.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def particle_generator():
     while True:
 
 #        if np.random.choice([0,1]):
-        stats_ = scpp.extract_nth_largest(stats,settings,n=index)
+        stats_ = scpp.extract_nth_largest(stats,n=index)
```
Comment (tankefugl) [inline tfl_tools/silcam_classifier_database_builder.py:43]: E231: missing whitespace after ','

**Review by tankefugl — CHANGES_REQUESTED** [review]
[Lintly](https://github.com/grantmcconnaughey/Lintly) has detected code quality issues in this pull request.

**Review by nepstad — APPROVED** [review]

**Review by nepstad — DISMISSED** [review]

File: tfl_tools/silcam_classifier_database_builder.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def particle_generator():
     while True:
 
 #        if np.random.choice([0,1]):
-        stats_ = scpp.extract_nth_largest(stats,settings,n=index)
+        stats_ = scpp.extract_nth_largest(stats, n = index)
```
Comment (tankefugl) [inline tfl_tools/silcam_classifier_database_builder.py:43]: E251: unexpected spaces around keyword / parameter equals

Comment (tankefugl) [inline tfl_tools/silcam_classifier_database_builder.py:43]: E251: unexpected spaces around keyword / parameter equals

**Review by tankefugl — DISMISSED** [review]
[Lintly](https://github.com/grantmcconnaughey/Lintly) has detected code quality issues in this pull request.

**Review by nepstad — APPROVED** [review]

**Review by WillRNaylor — APPROVED** [review]
Code looks good.

**Pull request merged**